test/cli/help/dbt-generate-profile/args

```diff
@@ -0,0 +1 @@
+dbt generate profile --help
```

Comment:
Bolo by super mat tu E2E testy na tie 4 situacie co som hovoril:
- subor existuje
- subor existuje, profile sa prepise
- subor neexistuje
- subor existuje, ale nie je validny YAML

Reply:
dobře

Reply:
(ak chces radsej robit `sources` command a testy doplnit neskor, tak moze byt aj tak, ale prakticky, to staci 4x nakopirovat a trocha upravit)

Reply:
V pohodě, nepřišlo mi to tak složitý aby byly třeba, ale není problém.

Reply:
https://github.com/keboola/keboola-as-code/pull/832/commits/406859614b4d891ecd33a5243b96f0137c551b08